### DISCUSSION TITLE
Welcome to Komunumo Discussions!

### DISCUSSION THREAD

**Discussion by McPringle** [McPringle/komunumo#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by das-g** [comment]
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Hi, I'm Raphael or @das-g

I'm getting more and more annoyed with the walled garden that Meetup.com is/has become. Thus when I saw https://fosstodon.org/@McPringle/114490618194917841 today (which probably got displayed to me due to the [`#TDD` hashtag](https://fosstodon.org/tags/TDD)), I got curious about this project. Even more so as someone recognizing its name as [the Esperanto word for "commune" and "community"](https://www.reta-vortaro.de/revo/dlg/index-2m.html?q=komunumo).
